.clang-format

```diff
@@ -3,7 +3,11 @@ BasedOnStyle: LLVM
 IndentWidth: 4
 UseTab: Never
 ColumnLimit: 100
-BreakBeforeBraces: Linux
+BreakBeforeBraces: Custom
+BraceWrapping:
+  AfterClass: false
+  AfterNamespace: false
+  AfterFunction: true
```

Comment:
This setting makes the following change

Before
```cpp
class MyClass {
public:
    void myFunction() {
        // function body
    }
};
```
After
```cpp
class MyClass {
public:
    void myFunction()
    {
        // function body
    }
};
```
This change is okay for `.cc` files, but it shouldn't apply to the header files.

Reply:
I would strongly suggest to use single format coding for all type of files. Do you prefer option ``.cc`` or ``.h`` ?

Reply:
We can consider the `Before` option for `.cc` files as well.

Reply:
> We can consider the `Before` option for `.cc` files as well.

Agree